libs/net/scripts/db-migration.sh

```diff
@@ -1,30 +1,36 @@
 #!/bin/bash
+if [ "$(uname)" = "Darwin" ]; then
+    sedcli="gsed"
+else
+    sedcli="sed"
+fi
+
```

Comment:
@Fosol let me know what you think about this -- trying to make it multiplatform compatible, please make sure it doesn't break anything on Windows!

Reply:
I like it.